### DISCUSSION TITLE
[Question]

### DISCUSSION THREAD

**Discussion by simple-AMS** [YetiForceCompany/YetiForceCRM#13962] [General]

**Comment by mariuszkrzaczkowski** [comment]
1 Please write questions in tab discussions >> https://github.com/YetiForceCompany/YetiForceCRM/discussions
2 I'm not sure if it's a questions or bug report
3 Version 6.1 without SP can works slower because in mail list we verify sender of a message. We recommend:
- install "YetiForce CRM 6.1 Service Pack".
- reduction a mails limit on the list

**Reply by simple-AMS** [reply]
Thanks @mariuszkrzaczkowski please see discussion #13965
